jwst/badpix_selfcal/badpix_selfcal_step.py

```diff
@@ -0,0 +1,197 @@
+
+
+import warnings
+from ..stpipe import Step
+from . import badpix_selfcal
+import numpy as np
+from jwst import datamodels as dm
+
+import logging
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
```

Comment:
I'm honestly not that familiar with the pipeline logging stuff but is this needed at the import level?

I think within the `Step` `self.log` can be used (it's set up [here](https://github.com/spacetelescope/stpipe/blob/d1a0a6d25de45400e315afa4fa3cee8633a6b545/src/stpipe/step.py#L366) in stpipe and appears to be used by other steps).

Perhaps @hbushouse can advise on which should be used.

Reply:
using `self.log` does seem to work identically, so going with @braingram suggestion